Source/Core/Common/Matrix.h

```diff
@@ -24,6 +24,12 @@ union TVec3
   {
   }
 
+  constexpr bool operator==(const TVec3& other) const
+  {
+    return this->x == other.x && this->y == other.y && this->z == other.z;
+  }
+  constexpr bool operator!=(const TVec3& other) const { return !(*this == other); }
```

Comment:
We use C++20 now, right? The `!=` operator is automatically synthesised using `operator==`, so defining `operator!=` manually is no longer necessary.

Reply:
Oh, is it? Great, I'll drop those then.